src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SnakemakeStatementParsing.kt

```diff
@@ -80,9 +114,17 @@ class SnakemakeStatementParsing(
         val scope = context.scope
 
         myBuilder.setDebugMode(false)
+        if (myBuilder.tokenType == PyTokenTypes.IDENTIFIER) {
+            // XXX: workaround/fix for #130:
+            val actualToken = SnakemakeTokenTypes.stringToTokenType[myBuilder.tokenText]
+            if (actualToken != null) {
+                myBuilder.remapCurrentToken(actualToken)
```

Comment:
What about filter method? I think it also should be updated somehow or even removed.